src/features/ContentTypes.feature

```diff
@@ -0,0 +1,161 @@
+Feature: Content types management
+  As a administrator
+  In order to customize my eZ installation
+  I want to manage my Content types structure.
+      
+  @javascript @common
+  Scenario: Content Type Groups context is available
+    Given that im in context of Content Type Groups
+    Then i can see Content Type Groups list
+      
+  @javascript @common
+  Scenario: Changes can be discarded while creating new Content Type Group
+    Given that im in context of Content Type Groups
+    When i click plus button
+      And fill Name field
+      And click Discard Changes
+    Then i shoud be redirected to Content Types context
+      And i should see Content Type Groups list
+      And there's nothing new on the list
+      
+  @javascript @common
+  Scenario: New Content Type Group can be added
+    Given that im in context of Content Type Groups
+    When i click plus button
+      And i fill Name field
+      And i click Create button
+    Then i should be in new group context
+      
+  @javascript @common
+  Scenario: Changes can be discarded while creating Content Type
+    Given that im in context of Content Type Group
+    When i click plus button
+      And i fill Name field
+      And i fill Identifier field
+      And i click Discard changes
+    Then i shoud be redirected to Content Type Group context
+      And i should see Content Types list
+      And there's nothing new on the list
+
+  @javascript @common
+  Scenario: New Field can be added to new Content Type
+    Given that im in context of Content Type Group
+    When i click plus button
+      And i fill Name field
+      And i fill Identifier field
+      And i fill Content name pattern field
+      And i select Text line content field definition
+      And Click Add field definition
+    Then message "Content type 'content_type_name' updated." appears
+      And form for new fieldtype definition appears.
+
+  @javascript @common
+  Scenario: New Content Type can be added to Content Type Group
+    Given that i'm in context of content type update form
+      And there are all necessary field filled
+      And there is at least one field added
+    When i click Save
+    Then i should be redirected to new Content Type context
+      And Content Type has proper values
```

Comment:
Could we specify the values/fields here, perhaps using a table? (http://docs.behat.org/en/v2.5/guides/1.gherkin.html#tables).
Something like:
```
And Content Type is properly saved:
| field | value |
| id    | val1  |
| name  | val2  |
```